crates/store/src/error.rs

```diff
@@ -0,0 +1,33 @@
+// SPDX-FileCopyrightText: 2022 Profian Inc. <opensource@profian.com>
+// SPDX-License-Identifier: AGPL-3.0-only
+
+use std::fmt;
+
+use crate::GetError;
+
+#[derive(Debug)]
+pub enum Error {
+    Io(std::io::Error),
+    Bincode(bincode::Error),
+    InvalidKeyFile,
+    BadMetaFile(String, Box<Error>),
+    BadBinFile(String, Box<Error>),
+    NotFound,
+}
```

Comment:
Do we have any code actually matching on these variants and handling them differently?
If all we want to do here is to report an error to the user, so why not just use a user-readable string (`&'static str`)?

Otherwise, https://docs.rs/anyhow/latest/anyhow/struct.Error.html is also an option, but I don't think we should be maintaining a separate error type for each module.